gtsam/linear/GaussianConditional.cpp

```diff
@@ -305,23 +305,36 @@ namespace gtsam {
     const Vector x =
         frontalValues.vector(KeyVector(beginFrontals(), endFrontals()));
 
-    // Copy the augmented Jacobian matrix:
-    auto newAb = Ab_;
+    // Compute updated right-hand side: d - R * x
```

Comment:
What prompted this change? `likelihood` is not used very much so it's not a performance bottleneck I think?

Reply:
I did this to avoid hitting the slow path in `unweighted_error` as this used to be using `firstBlock`